core/src/main/java/com/datastax/oss/driver/api/core/cql/ExecutionInfo.java

```diff
@@ -34,9 +36,11 @@
 public interface ExecutionInfo {
 
   /** The statement that was executed. */
+  @NonNull
   Statement<?> getStatement();
 
   /** The node that was used as a coordinator to successfully complete the query. */
+  @NonNull
   Node getCoordinator();
```

Comment:
This can actually be null under certain circumstances. My bad because I should have documented it in the first place, I will push a commit to address it.

Reply:
Nevermind, that's only after my changes in #1038